src/blockchain.rs

```diff
@@ -371,18 +371,32 @@ impl Blockchain {
             " ... blockchain.add_block_success: {:?}",
             create_timestamp()
         );
-        let block_id;
+        // print blockring longest_chain_block_hash infor
+        self.print();
 
         //
         // save to disk
         //
         {
             let block = self.get_mut_block(&block_hash).await;
-            block_id = block.get_id();
             if block.get_block_type() != BlockType::Header {
                 Storage::write_block_to_disk(block);
             }
         }
+
+        //
+        // TODO: clean up mempool - I think we shouldn't cleanup mempool here.
```

Comment:
To have an overview about open TODOs we tend to create tickets and link those in the code. Just an idea. /cc @karlsaito